packages/calling-component-bindings/src/videoGallerySelector.ts

```diff
@@ -114,7 +114,7 @@ const dominantSpeakersWithFlatId = (dominantSpeakers?: DominantSpeakersInfo): un
  */
 const sortedRemoteParticipants = (
   participants?: VideoGalleryRemoteParticipant[],
-  dominantSpeakers?: string[]
+  dominantSpeakers?: Record<string, number>
```

Comment:
`<string, number>` is difficult to know what the types are, in the function can you mention this is userId -> dominantSpeakerRank
Or better yet do we have types for these? e.g. `Record<flatCommunicationIdentifier, dominantSpeakerRank>`

Reply:
All this code will go away in the next PR for new layouts in video gallery :)